front/src/components/Feed.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components';
+// import {} from 'styled-components/cssprop';
+import { ReactComponent as VertBtnSvg } from '../assets/icons/more_vert_btn.svg';
+import { ReactComponent as HeartBtnSvg } from '../assets/icons/empty_heart_btn.svg';
+import { ReactComponent as CommentBtnSvg } from '../assets/icons/comment_btn.svg';
+
+export type t_FeedJson = {
+  id: number;
+  nickname: string;
+  imageURL: string;
+  text: string;
+};
+
+const FeedContainer = styled.div<{ imageURL: string }>`
```

Comment:
```suggestion
    const FeedContainer = styled.div<Pick<t_FeedJson, 'imageURL'>>`
```
`imageURL`라는 타입은 이미 `t_FeedJson` 타입에 정의되어 있으니 `Pick`을 이용해서 `imageURL` 타입만 빼와서 재사용할 수 있지 않을까요?

근데 `styled` 문법에서 제대로 돌아가는지는 모르겠네요 ㅋㅋ

Reply:
오 몰랐던 문법인데 좋은 것 같습니다. 타입의 하위 집합을 사용할때는 Pick을 사용하도록 컨벤션으로 강제하는 것도 좋을 것 같아요

Reply:
저도 몰랐던 문법인데 좋네요